packages/apps/web-viewer-test/tsconfig.json

```diff
@@ -13,8 +17,10 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx",
+    "jsx": "react",
```

Comment:
isn't react-jsx the new value (React 17)?

Reply:
this is one of those changes made by cra, ill revert

Reply:
<img width="596" alt="image" src="https://user-images.githubusercontent.com/11051042/179825715-70fac4aa-a4cf-4168-9cd7-32e3c4ad1fd6.png">

actually looks like this is the right one we want for react 17